signal_analog/util.py

```diff
@@ -23,6 +25,22 @@ def in_given_enum(value, enum):
         raise ValueError(msg.format(value, valid_values))
 
 
+def check_markdown(text, error_message=None):
+    """Determines if the given text is a markdown and able to generate HTML
+    """
+    if text is '' or text is None:
+        if error_message:
+            raise ValueError(error_message)
+        else:
+            raise ValueError()
+    try:
```

Comment:
I don't understand what catching the Error and immediately raising it is doing.  Can you explain?  Thanks

Reply:
I thought of catching UnicodeDecodeError and raise ValueError. Felt unnecessary, so I left it for feed back.

Reply:
Thinking about this more I think I would rather leave the unicode error rather than re-throwing as `ValueError`. It gives the user a bit more context as to what the error was. I'm open to discussing this point more though if there's a strong case to leave it as is!